beacon-chain/node/helper.go

```diff
@@ -0,0 +1,49 @@
+package node
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// Given input string `block_root:epoch_number`, this verifies the input string is valid, and
+// returns the block root as bytes and epoch number as unsigned integers.
+func convertWspInput(wsp string) ([]byte, uint64, error) {
+	if wsp == "" {
+		return nil, 0, nil
+	}
+
+	// Weak subjectivity input string must contain ":" to separate epoch and block root.
+	if !strings.Contains(wsp, ":") {
+		return nil, 0, fmt.Errorf("%s did not contain column", wsp)
+	}
+
+	// Strip prefix "0x" if it's part of the input string.
+	if strings.HasPrefix(wsp, "0x") {
+		wsp = wsp[2:]
+	}
+
+	// Get the hexadecimal block root from input string.
+	s := strings.Split(wsp, ":")
+	if len(s) != 2 {
+		return nil, 0, errors.New("bad format string")
+	}
```

Comment:
Maybe output why it is not well formatted, showing them the expected format. Perhaps that should also be part of the flag usage if not specified already

Reply:
it's part of the flag usage. I'll reiterate in the error again